source/framework/vx_parameter.c

```diff
@@ -344,6 +344,32 @@ VX_API_ENTRY vx_status VX_API_CALL vxSetParameterByIndex(vx_node node, vx_uint32
                 }
             }
 
+            /* Check for node replication; ref should be first element of an object array or pyramid */
```

Comment:
Gentle ping on this

Reply:
is it a question for Bosch?

Reply:
@raphacan Yes--it looked like this section of code was a separate bug fix than the supplementary data feature.  I thought it could make sense to be added separately as a bug fix commit which could be pulled in sooner than the rest of the extension if my understanding of this section of code was correct.